README.md

```diff
@@ -149,6 +149,16 @@ const [gql, resolver] = usecase2mutation(usecase, resolverFunc)
 ```
 ​
 Or you can use `herbs2gql` [`defaultResolver`](https://github.com/herbsjs/herbs2gql/blob/master/src/defaultResolver.js) implementation as a reference. 
+
+### Herbarium integration
```

Comment:
>Herbarium integration

Since this kind of integration will be the primary way of converting Herbs to GraphQL, this should be the first method to show, not the last

Reply:
make sense, fixed